packages/expo-contacts/ios/ContactAccessPicker.swift

```diff
@@ -0,0 +1,61 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+import SwiftUI
+import ContactsUI
+import ExpoModulesCore
+
+/**
+ A plain view that presents contact access picker on mount.
+ */
+@available(iOS 18.0, *)
+internal struct ContactAccessPicker: View {
+  @State
+  private var isPickerPresented = false
+
+  internal var completion: (([String]) -> Void)?
+
+  var body: some View {
+    Rectangle()
```

Comment:
I'll replace it with `EmptyView` before merging, just don't want to trigger CI once again